.github/workflows/labeler.yml

```
@@ -0,0 +1,29 @@
+  label:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event.pull_request.head.ref }}
+          repository: ${{ github.event.pull_request.head.repo.full_name }}
+          token: ${{ secrets.GITHUB_TOKEN }}
+      - uses: actions/labeler@v5
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          issue-types: "labeled,unlabeled"
```

actionlint (1.7.12)

[error] input "issue-types" is not defined in action "actions/labeler@v5". available inputs are "configuration-path", "dot", "pr-number", "repo-token", "sync-labels"

(action)